templates/react/.gitignore

```diff
@@ -1,2 +0,0 @@
-.idea
```

Comment:
sth is no yes here, you are not working in this directory, you shouldn't edit files in this directory, again could be rebase problem